src/IO/InterfaceImpl.cpp

```diff
@@ -65,8 +65,7 @@ void InterfaceImpl::initializeOutputs()
 #endif
 }
 
-bool InterfaceImpl::updateInputs()
-{
+bool InterfaceImpl::updateInputs() {
```

Comment:
Undo this, current formatting is to put the bracket on the next line

Reply:
We really need to get a common setup for linter and formatting